seimas/accounts/forms.py

```diff
@@ -12,3 +12,10 @@ class Meta:
                 "Bus naudojamas komunikacijai. Jei pageidaujate negauti jokių laiškų, palikite šį lauką tuščią."
             ),
         }
+
+
+class LoginForm(forms.ModelForm):
+    class Meta:
+        model = auth_models.User
+        fields = ('username', 'password')
+        
```

Comment:
Use [built-in Django forms](https://docs.djangoproject.com/en/1.9/topics/auth/default/#django.contrib.auth.forms.AuthenticationForm) instead of creating your own.